_docker/drupal/drupal-filesystem/web/config/sync/rhd_disqus.disqussettings.yml

```diff
@@ -0,0 +1,6 @@
+rhd_disqus: null
+_core:
+  default_config_hash: xcRoj14zprIFuKVJB0s_N6BC9tOPuD1McpaYgXd8Hlo
+rhd_disqus_api_key: gv1imdagtANIODWhIySfgTwFBMvdq8KrtW2P1IxvStvXfbqM9STyvhaKypUT4XWa
+rhd_disqus_shortname: redhat-developers-blog
+rhd_disqus_secret_key: lybYUvcBJmpgvAziyAz2qzukFrDw4eQ2WJJFvjVqHjXhS8tcCEOMiFhSMsoaLiNv
```

Comment:
Uh.... am I the only one seeing the MASSIVE security hole of storing this stuff here on the file system which is then pushed (is CURRENTLY pushed) to PUBLIC GitHub?

Reply:
These have been changed. When the module uses database storage, you can request new creds.

Reply:
I understand the concern with storing this in a public GitHub. What are your preferences for dealing with config-stored keys like this?

Reply:
Thanks, Kerri!